spec/std/big/big_float_spec.cr

```diff
@@ -228,8 +228,13 @@ describe "BigFloat" do
   end
 
   it "#hash" do
+    a = -123.to_big_f
+    a.hash.should eq BigFloat.new("-123").hash
+
     b = 123.to_big_f
-    b.hash.should eq(b.to_f64.hash)
+    b.hash.should eq(b.to_s.hash)
```

Comment:
If you're breaking this spec, just delete it. Adding this line is disingenuous, you're just asserting the exact same thing that the implementation does, it's not an appropriate replacement.

Reply:
That's exactly what previous assertions was doing...

Reply:
Hmm right.

Should have been written as `b.hash.should eq(123.0.hash)`